README.md

```diff
@@ -57,16 +20,55 @@ See [Support Site]() for EFSA developer details.
 - Clone this repository
 - Open Visual Studio as an administrator
 
-##### Publish Database
+#### Publish Database
 
 - Build the solution SFA.DAS.ProviderApprenticeshipService.sln
-- Either use Visual Studio's `Publish Database` tool to publish the database project SFA.DAS.ProviderAgreementStatus.Database to name {{database name}} on {{local instance name}}
+- While running Visual studio in Administrator Mode: Either use Visual Studio's `Publish Database` tool to publish the database project SFA.DAS.ProviderAgreementStatus.Database with name **SFA.DAS.ProviderAgreementStatus.Database** on **ProjectsV13** (or whatever local instance you are using)
 
 	or
 
 - Create a database manually named {{database name}} on {{local instance name}} and run each of the `.sql` scripts in the SFA.DAS.ProviderApprenticeshipService.Database project.
 
-##### Config
+#### PAS Seed Data
+
+Insert the following record into the ProviderAgreementStatus database in order to simulate the given Provider (in this case, _10005077_) having signed their agreement with the SFA. This unlocks all of the approval functionality.
+
+```SQL
+insert into ContractFeedEvent (Id, ProviderId, HierarchyType, FundingTypeCode, [Status], ParentStatus, UpdatedInFeed, CreatedDate)
+values (NEWID(), '10005077', 'CONTRACT', 'LEVY', 'APPROVED', 'APPROVED', GETDATE(), GETDATE())
+```
+
+## Configuration
+
+#### Automatically obtain config (Recommended)
+
+- Use the [das-employer-config-updater](https://github.com/SkillsFundingAgency/das-employer-config-updater) to obtain the latest config
+
+## Running the solution
+- Ensure you have the SFA.DAS.Commitments project set up (follow its readme file)
+- Open the solution SFA.DAS.ProviderApprenticeshipService.sln as **administrator**
+- Open the solution SFA.DAS.Commitments.sln as **administrator**
+- Close all instances of "Microsoft Azure storage emulator" within the system tray
+
+- Set SFA.DAS.ProviderApprenticeshipService startup projects to 'Multiple startup projects'. The **only** projects set to start should be: 
+  - SFA.DAS.API (PAS Account API)
+  - SFA.DAS.CloudService (PAS Web UI)
+  
+- Clean the SFA.DAS.ProviderApprenticeshipService solution
+- Rebuild the SFA.DAS.ProviderApprenticeshipService solution
+
+- Run the SFA.DAS.Commitments.sln (this should automatically start the Microsoft Azure storage emulator), it normally starts with an error page. You can check if it's running by calling the "/api/healthcheck" endpoint which should return status code 200.
+
+- Run the SFA.DAS.ProviderApprenticeshipService.sln, it may several minutes to start the first time.
+
+- Once both solutions are fully running, open an incognito tab in any browser and navigate to the "misc helpers" website (Get link from Approvals / Continous Improvement team)
```

Comment:
why cant we put the link directly in here?

Reply:
I noticed it contains encoder/decoder, although it apparently doesn't work, I've never tested it myself